.circleci/dynamic_config.yml

```diff
@@ -291,19 +295,39 @@ jobs:
       - custom_attach_workspace
       - run: yarn bazel build //tests/legacy-cli/...
 
-  test:
+  unit-test:
     executor: test-executor
     resource_class: xlarge
+    parameters:
+      nodeversion:
+        type: string
+        default: *default_nodeversion_major
     steps:
       - custom_attach_workspace
       - browser-tools/install-chrome
       - setup_bazel_rbe
       - run: sudo cp .circleci/bazel.rc /etc/bazel.bazelrc
-      - run:
-          command: yarn bazel:test
-          # This timeout provides time for the actual tests to timeout and report status
-          # instead of CircleCI stopping the job without test failure information.
-          no_output_timeout: 40m
+      - when:
+          # The default nodeversion runs all *excluding* other versions
+          condition:
+            equal: [*default_nodeversion_major, << parameters.nodeversion >>]
+          steps:
+            - run:
+                command: yarn bazel test --test_tag_filters=-node16,-node<< parameters.nodeversion >>-broken //packages/...
```

Comment:
This ugly condition-x + condition-not-x is to ensure no tests are forgotten.

When running the `default_nodeversion_major` it runs all tests *excluding* other versions.
When running other version it runs all tests *including* that specific version.

If the `node{{nodeversion}}` tag is ever forgotten it will be included in the first case. Where if we always do the *including* (and got rid of this ugly condition) then forgetting a tag would cause the test to never run.